Cookbook/Technical-Documents/TaskReviewGuidelines.md

```diff
@@ -0,0 +1,60 @@
+
+# Technical test reviewer guidelines
+
+This document describes the guidelines for reviewing technical task assignments that candidates work on during the recruitment process.
+Every question is attributed with **S/M/J** or a combination of those, defining what level it relates to.
+
+- **S** (expected of a senior-level task)
+- **M** (expected of a mid-level task)
+- **J** (expected of a junior-level task)
+
+## General:
+
+### Basics
+- Does it have clear instructions on how to run the project? (SMJ)
+- Are there any compile time issues? (SMJ)
+- Does it meet functional requirements? (SMJ)
+- Are there any run time issues? (SM)
+- Does it have descriptive README? (SM)
+
+### Good practices
+
+- Is it testable? (S, some for M)
+- Does it conform to the SOLID principles? (S, some for M)
```

Comment:
How do we quantify `some` here? Is there a minimum amount expected, and are there varying amounts of this quantity for other points throughout this document?

Reply:
@scass91  What we mean by `Some` is that an ideal Senior candidate's tech test should have all parts of the app designed for testability, but for a Mid level candidate, we can consider if they exhibit this in a few parts. It is hard to make every part of the app testable, which is why this differentiation. If there are better ways to word this suggestions welcome.

Reply:
Ahhh ok so there's no real quantity expected, just a usage in *some* cases where it can be used? That makes sense - thanks.

Reply:
I think its better we add a description of what `some` means in the beginning to make it clearer. @konrad-em Thoughts?

Reply:
I agree, on second thought I find `some` in this particular case (SOLID) a bit vague. But does it need more explanation in other questions? Let's keep in mind these are rather guidelines than strict step by step instructions and we've intentionally left some space for interpretation :)